### ISSUE TITLE
v4 Language Picker is not accessible with the keyboard

### ISSUE THREAD

**Issue by RobertPeek** [adaptlearning/adapt_framework#2941]
I cannot tab to the language buttons in the language picker menu when the course loads.

### Environment
* FW version 4.5.0
* Language Picker version 3.0.2

This issue does seem to be resolved in FWv5 and the latest Language picker version but unfortunately I'm still having to use FWv4.

**Comment by oliverfoster** [comment]
closed as stale - known issue
